package.json

```diff
@@ -107,7 +107,6 @@
         "electron": "30.3.1",
         "@types/node": "20.12.7",
         "@types/react": "18.2.55",
-        "@sentry/types": "7.92.0",
```

Comment:
I just removed this resolution and typescript wasn't complaining, so I guess it's not needed anymore.